lib/layout.js

```diff
@@ -215,8 +215,10 @@ El.calcSize = (el) => (
 
 El.calcScrollSize = (el) => (
   el === window
-    ? { w: el.scrollX, h: el.scrollY }
-    : { w: el.scrollLeft, h: el.scrollTop }
+   ? {
+    w: el.scrollX || el.pageXOffset || document.documentElement.scrollLeft,
+    h: el.scrollY || el.pageYOffset || document.documentElement.scrollTop
```

Comment:
makes total sense :) I'll make the changes

On Thu, Sep 3, 2015 at 4:36 PM, Jason Rose-Kuhrt <notifications@github.com>
wrote:

> In lib/layout.js
> <https://github.com/littlebits/react-popover/pull/32#discussion_r38652447>
> :
>
> > @@ -215,8 +215,10 @@ El.calcSize = (el) => (
> >
> >  El.calcScrollSize = (el) => (
> >    el === window
> > -    ? { w: el.scrollX, h: el.scrollY }
> > -    : { w: el.scrollLeft, h: el.scrollTop }
> > +   ? {
> > +    w: el.scrollX || el.pageXOffset || document.documentElement.scrollLeft,
> > +    h: el.scrollY || el.pageYOffset || document.documentElement.scrollTop
>
> I misread my own function pardon me; right we are resolving the check for
> window in this case not el.
>
> This component uses flexbox and inline svg so unless you can find a
> browser that supports those but not pageX/YOffset I am asking for that
> third fallback to be removed. Of course I'm happy to accept the
> pageX/YOffset fallback though. Make sense?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/littlebits/react-popover/pull/32/files#r38652447>.
>